third_party/Makefile

```diff
@@ -3,27 +3,18 @@ include ../config.mak
 ARCH ?= x86_64
 
 .PHONY: clean distclean
-.INTERMEDIATE: ${MAKE_ROOT}/cryptsetup.tar.gz ${MAKE_ROOT}/devicemapper.tar.gz ${MAKE_ROOT}/e2fsprogs.tar.gz ${MAKE_ROOT}/util-linux.tar.gz ${MAKE_ROOT}/popt.tar.gz ${MAKE_ROOT}/json-c.tar.gz ${MAKE_ROOT}/wireguard.tar.gz\
-               ${CRYPTSETUP} ${DEVICEMAPPER} ${UTILLINUX} ${POPT} ${JSONC} ${WIREGUARD}
+.INTERMEDIATE: ${MAKE_ROOT}/devicemapper.tar.gz ${MAKE_ROOT}/e2fsprogs.tar.gz ${MAKE_ROOT}/json-c.tar.gz ${MAKE_ROOT}/wireguard.tar.gz\
```

Comment:
What is json-c used for now?

Reply:
That's a good point. @wintersteiger can that also be removed now?

Reply:
Yes, cryptsetup was the last bit that depended on it, go ahead and remove it!

Reply:
Removed.